src/GraphEditor.js

```diff
@@ -13,7 +13,14 @@ import {displayGraphType} from './utils';
 
 export default class GraphEditor extends React.Component {
     title() {
-        return <h2>{displayGraphType(this.props.gType)}</h2>;
+        return (
+            <div>
+            <h1>{displayGraphType(this.props.gType)}</h1>
+            <p className="lead text-secondary">
+                Add and modify the information of your graph.
+            </p>
+        </div>
```

Comment:
btw - [React.Fragment](https://reactjs.org/docs/fragments.html) Allows you to return a list of elements without having to surround them in a div. In this case, maybe it makes sense to have them in their own div though.